scanpy/tools/_umap.py

```diff
@@ -1,5 +1,5 @@
 from ._utils import get_init_pos_from_paga, choose_representation
-from .. import settings
+from .._settings import settings
```

Comment:
This should not have been necessary. It should be `from .. import settings`...

Reply:
It is the same as in the master branch
https://github.com/theislab/scanpy/blob/9b522f54e0f839e1a0c9874ca658400bfe79a894/scanpy/tools/_umap.py#L2

Reply:
Well, it doesn't hurt, I don't who put it there. But it should be `from ..settings import settings`. :) Don't worry...